docs/integrations/data-ingestion/clickpipes/mysql/faq.md

```diff
@@ -9,10 +9,15 @@ title: 'ClickPipes for MySQL FAQ'
 # ClickPipes for MySQL FAQ
 
 ### Does the MySQL ClickPipe support MariaDB?  {#does-the-clickpipe-support-mariadb}
-Yes, the MySQL ClickPipe supports MariaDB 10.0 and above. The configuration for it is very similar to MySQL, with the GTID behaviour being enabled by default.
+Yes, the MySQL ClickPipe supports MariaDB 10.0 and above. The configuration for it is very similar to MySQL, using GTID replication by default.
 
-### Does the MySQL ClickPipe support Planetscale, Vitess? {#does-the-clickpipe-support-planetscale-vitess}
-Currently, we support only standard MySQL. Since PlanetScale is built on Vitess, integrating with Vitess's VStream API and handling VGtids (Vitess Global Transaction IDs) to track incremental changes is required. This differs from how CDC operates in native MySQL. Adding support for this functionality is being actively worked on.
+### Does the MySQL ClickPipe support PlanetScale, Vitess, TiDB? {#does-the-clickpipe-support-planetscale-vitess}
+No, these do not support MySQL's binlog API.
```

Comment:
I did an evaluation of these alternative's CDC story. Vitess CDC lacks authentication (ie, you connect to grpc endpoint & have complete access). As a result PlanetScale made their own CDC protocol which we would have to implement based on their airbyte connector as a reference implementation. TiDB deprecated support for binlog API, instead implementing [TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview)

This fractured landscape makes implementing a connector for these services less appetizing, to the point that we are not actively working on any of them